query-node/schemas/contentNftEvents.graphql

```diff
@@ -530,6 +530,36 @@ type BuyNowCanceledEvent implements Event @entity {
   ownerCuratorGroup: CuratorGroup
 }
 
+type BuyNowPriceUpdatedEvent implements Event @entity {
+  ### GENERIC DATA ###
+
+  "(network}-{blockNumber}-{indexInBlock}."
+  id: ID!
+
+  "Hash of the extrinsic which caused the event to be emitted."
+  inExtrinsic: String
+
+  "Blocknumber of the block in which the event was emitted."
+  inBlock: Int!
+
+  "Network the block was produced in."
+  network: Network!
+
+  "Index of event in block from which it was emitted."
+  indexInBlock: Int!
+
+  ### SPECIFIC DATA ###
+
+  "NFT's video."
+  video: Video!
+
+  "Content actor acting as NFT owner."
+  contentActor: ContentActor!
+
+  "New buy-now price."
+  newPrice: BigInt!
+}
```

Comment:
Look good. Just one thing do we need to add NFT owners fields here?

```javascript
  "Member owning the Nft (if any)."
  ownerMember: Membership

  "Curator group owning the Nft (if any)."
  ownerCuratorGroup: CuratorGroup
```

Reply:
Added.